src/gaming/ExtendedNFT721Manager.ts

```diff
@@ -10,6 +10,11 @@ import { NFT721Manager } from '../managers'
 export class ExtendedNFT721Manager extends NFT721Manager {
   protected _ipfs: Ipfs | undefined
 
+  constructor () {
+    super()
+    console.warn("ExtendedNFT721Manager is deprecated. Use NFT721Manager instead.")
```

Comment:
'Deprecation notice: the ExtendedNFT721Manager class is being deprecated. Use NFT721Manager instead.'
also for the Token20Manager